lib/rails_erd/domain/attribute.rb

```diff
@@ -125,7 +125,7 @@ def type_description
       # Returns any non-standard limit for this attribute. If a column has no
       # limit or uses a default database limit, this method returns +nil+.
       def limit
-        return if native_type == 'geometry'
+        return if native_type == 'geometry' || native_type == 'geography'
```

Comment:
If we add a third, I'll switch this to a constant like `EXCLUDED_TYPES` and check inclusion